src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -28877,6 +29417,9 @@ static void M1<T>(T t1)
 }";
             var comp = CreateCompilation(new[] { DisallowNullAttributeDefinition, source }, options: WithNonNullTypesTrue());
             comp.VerifyDiagnostics(
+                // (11,12): warning CS8607: A possible null value may not be assigned to a target marked with the [DisallowNull] attribute
```

Comment:
nit: Consider updating the diagnosic message in this comment. I had started to comment about how this message could be better but it turned out you already changed it for the better :)